design-library/src/components/BccTabs/BccTabs.stories.ts

```diff
@@ -22,6 +22,11 @@ export default {
       options: ["sm", "base", "lg"],
       control: { type: "radio" },
     },
+    //Boolean fill
+    fill: {
+      description: "Fill up the entire space of the parent flex container",
```

Comment:
Looking at this description I'm not entirely sure what this prop does. It fills it with a background color and applies sticky as well. From this description I would just expect it to stretch to full-width?

Reply:
It should fill up both width and height.

Reply:
People can also view what it does via stories.

Reply:
> People can also view what it does via stories.

That's the thing, the only thing it seems to do when toggling `fill` is to set a dark grey background, which feels a bit unexpected to me 😅 I would expect it only changes the width/height.

Reply:
RIght, that dark background actually comes from the design system, I did speak to Kaja about it, but didn't land on anything better yet.